rust/azure_iot_operations_protocol/src/telemetry/sender.rs

```diff
@@ -189,6 +189,9 @@ pub struct Message<T: PayloadSerialize> {
     /// Cloud event of the telemetry message.
     #[builder(default = "None")]
     cloud_event: Option<CloudEvent>,
+    /// Indicates whether the message should be retained or not.A retained message is a normal MQTT message with the retained flag set to true.
```

Comment:
Do we want to mention MQTT here?

Reply:
There are some docs like message expiry which mentions MQTT while others don't. I have taken out the entire second line as the first line of doc is sufficient.